### ISSUE TITLE
BRE Pipeline Framework not work with vs2015

### ISSUE THREAD

**Issue by ZehBrz** [mbrimble/brepipelineframework#2]
Guys ... good mornig. I need orientation. I try install the msi 1.6 version from folder Release for use with Visual Studio 2015 Community Version and BizTalk Developer Edition 2016 and the BREPipelineFramework not show in Choose Items, tab BizTalk pipeline Components for add in visual studio toolbox for pipelines.

Where i fail in instalation process of msi file?

Tks in advanced.

**Comment by Dijkgraaf** [comment]
It is not yet compatible with BizTalk 2016.  This is being looked at, but no ETA as to when that will be completed.

**Comment by ZehBrz** [comment]
tks for your answer Dijkgraaf ... You have news about the changes in BTS2016 that impact BREPipelineFramework for BTS2013R2?

**Comment by RobBowman** [comment]
I am currently using the pipeline framework with BizTalk 2016 FP3 CU5, it is working ok. Was something fixed?

**Comment by Dijkgraaf** [comment]
@RobBowman   No, we finally got around to trying it out with BizTalk 2016 and it worked.  But I think we recompiled it in Visual Studio 2015 before deploying it rather than using the MSI for earlier versions of BizTalk.  Is that what you did?
The only issue we came across was [BizTalk 2016 Microsoft.BizTalk.Interop.SSOClient Assembly Version](https://www.connected-pawns.com/2018/10/26/biztalk_2016_microsoft_biztalk_interop_ssoclient/)
